schematicexport.py

```diff
@@ -3,22 +3,34 @@
 import os.path
 import re
 
+from pcbnew import (
+    GetBuildVersion,
+)
+
+from .helpers import is_nightly
 
 class SchematicExport:
 
     """A class to export Schematic files"""
 
-    """This only works with KiCad V6 files, if the format changes, this will probably break"""
+    """This only works with KiCad V6/V7 files, if the format changes, this will probably break"""
 
     def __init__(self, parent):
         self.logger = logging.getLogger(__name__)
         self.parent = parent
 
     def load_schematic(self, paths):
-        for path in paths:
-            self._update_schematic(path)
+        if is_nightly(GetBuildVersion()):
```

Comment:
Is function `is_nightly` the correct way of checking for KiCad 7? I am not a KiCad plugin developer, but I would think of something like this:
```
if GetBuildVersion().startswith('7'):
```

Reply:
currently "is_nightly" returns true for version 6.99, 7, or 7.99. It uses elsewhere for version 7 checking.

Reply:
Thank you for the response and explanation, I see. I just think it will be less confusing, if the function name will be `is_kicad7` or something similar that describes what the function really does.

Reply:
The `is_nightly` function was introduced back when there was only a nightly V6.99 which contained breaking changes.
So the name is no longer meaningful, I think I should come up with a smarter way of handling the different versions.